build_container/build_container_windows.ps1

```diff
@@ -75,15 +75,15 @@ mkdir -Force C:\tools
 # Bazelisk
 mkdir -Force C:\tools\bazel
 DownloadAndCheck C:\tools\bazel\bazel.exe `
-                 https://github.com/bazelbuild/bazelisk/releases/download/v1.5.0/bazelisk-windows-amd64.exe `
-                 67149e87d51eb2b34d8b22ee0aa4ae63550919d3f2792f863eaabf9e78826a60
+                 https://github.com/bazelbuild/bazelisk/releases/download/v1.7.4/bazelisk-windows-amd64.exe `
```

Comment:
can you update in linux image as well?

Reply:
Done in https://github.com/envoyproxy/envoy-build-tools/pull/113/commits/0977c3fd798911c7af591ae1f31b6c5968c2c5cf